lib/plugins/inspec-parallel/lib/inspec-parallel/command.rb

```diff
@@ -4,45 +4,66 @@
 
 module InspecPlugins
   module Parallelism
+
+    class OptionFileNotReadable < RuntimeError
+    end
+
     class Command
-      attr_accessor :options, :default_profile, :sub_cmd, :option_lines
+      attr_accessor :cli_options_to_parallel_cmd, :default_profile, :sub_cmd, :invocations
 
-      def initialize(options, default_profile, sub_cmd = "exec")
+      def initialize(cli_options_to_parallel_cmd, default_profile, sub_cmd = "exec")
         @default_profile = default_profile
-        @options = options
+        @cli_options_to_parallel_cmd = cli_options_to_parallel_cmd
         @sub_cmd = sub_cmd
         @logger  = Inspec::Log
-        @validation_passed = true
-        @option_lines = read_options_file
+        @invocations = read_options_file
       end
 
       def run
-        Runner.new(option_lines, options, sub_cmd).run
+        validate_invocations!
```

Comment:
Just curious, why are we validating invocation before run also now?

Reply:
Because validation needs to check (and possibly modify) the --reporter option to ensure that a child-status reporter is present.